weblate/templates/list-translations.html

```diff
@@ -65,6 +67,15 @@
 {{ suggestions }}
 {% endif %}
 </td>
+<td class="number col-comments">
+{% if trans.stats.comments > and translate_url %}
```

Comment:
Looks like 0 is missing ater > here.

Reply:
See error in tests: `django.template.exceptions.TemplateSyntaxError: Not expecting 'and' in this position in if tag.`